solidity/contracts/vault/DonationVault.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+
+pragma solidity ^0.8.9;
+
+import "./IVault.sol";
+import "../bank/Bank.sol";
+
+/// @title BTC donation vault
+/// @notice Vault that allows making BTC donations to the system. Upon deposit,
+///         this vault does not increase depositors' tBTC balances and always
+///         decreases its own tBTC balance in the same transaction.
```

Comment:
```suggestion
/// @notice Vault that allows making BTC donations to the system. Upon deposit,
///         this vault does not increase depositor's balances and always
///         decreases its own balance in the same transaction.
```

TBTC is only relevant for `TBTCVault`. All other system components operate on a balance in the Bank.

Reply:
See 2efa4ced